openreview/journal/invitation.py

```diff
@@ -211,6 +211,10 @@ def set_reviewer_recruitment_invitation(self):
             process_content = process_content.replace("ACTION_EDITOR_ACCEPTED_ID = ''", f"ACTION_EDITOR_ACCEPTED_ID = '{reviewers_id}'")
             process_content = process_content.replace("ACTION_EDITOR_DECLINED_ID = ''", f"ACTION_EDITOR_DECLINED_ID = '{reviewers_declined_id}'")
             process_content = process_content.replace("HASH_SEED = ''", f"HASH_SEED = '{self.journal.secret_key}'")
+            if self.journal.get_request_id():
+                process_content = process_content.replace("JOURNAL_REQUEST_ID = ''", "JOURNAL_REQUEST_ID = '" + self.journal.get_request_id() + "'")
+            process_content = process_content.replace("SUPPORT_GROUP = ''", f"SUPPORT_GROUP = 'openreview.net/Support'")
```

Comment:
@melisabok what would be the best way to get the value for `SUPPORT_GROUP` here? We don't really need it to find the recruitment notes but we do for the readers of the comment

Reply:
What about getting the request comment invitation and read the readers from there?